controllers/stargate/stargate_controller.go

```diff
@@ -215,6 +215,15 @@ func (r *StargateReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		}
 	}
 
+	// The following check makes sure that when Stargate is deployed on its own that the
+	// first deployed pod reaches the ready state before deploying more in order to avoid
+	// concurrent schema updates which Cassandra doesn't do well with. See
+	// https://github.com/k8ssandra/k8ssandra-operator/issues/297 for details.
+	if _, found := stargate.Labels[k8ssandraapi.K8ssandraClusterNameLabel]; !found && len(actualDeployments.Items) > 0 &&
```

Comment:
Shouldn't we create a new function in the `labels` package for that? Similar to `IsCreatedByK8ssandraController` but without considering the actual k8c name and namespace.